pynxtools_apm/config/eln_cfg.py

```diff
@@ -86,7 +86,7 @@
 
 APM_INSTRUMENT_STATIC_TO_NEXUS = {
     "prefix_trg": "/ENTRY[entry*]/measurement/instrument",
-    "prefix_src": "atom_probe/",
+    "prefix_src": "instrument/",
```

Comment:
Is the definition also updated?

Reply:
It does not need an update because prefix_src is the prefix for the source side which in this case is eln_data.yaml, the respective NOMAD OASIS ELN template has been updated,
if the change would be on the prefix_trg side, indeed an update on nexus_definitions would have been required.